### ISSUE TITLE
[TFS] Enable ProjectCop in build process

### ISSUE THREAD

**Issue by LianwMS** [OData/odata.net#586]
This is an issue for build process in TFS branch, For V7 branch, project cop is disabled to unblock build. It should be enabled later

**Comment by habbes** [comment]
Closing due to inactivity. If the issue still exists in the latest version of the library, kindly open a new issue.
